libs/utils/exception/src/lib/messages/dev.th.ts

```diff
@@ -79,27 +79,28 @@ export const DevErrorMessages = {
   '3005': {
     statusCode: '400',
     code: '3005',
-    message: 'payload กับ channel ไม่ตรงกัน'
+    message: 'ขออภัย มีบางอย่างผิดพลาด กรุณาลองใหม่อีกครั้ง'
```

Comment:
should be the same according to
https://docs.google.com/spreadsheets/d/1-ScxqntTscqUucOA5a2va5sZm42CntgeG8emmtWtpl8/edit#gid=286412981

Reply:
fixed